react-projects/src/components/Exercesise3.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Exercesise2 from './Exercesise2'
+
+const Exercesise3 = () => {
+  return (
+<div className=" flex justify-around ">
+        <Exercesise2 id="1"/>
+        <Exercesise2 id="2"/>
+        <Exercesise2 id="3"/>
+    </div>
+  )
```

Comment:
You have to remove duplication of components and naming is wrong change the name of component